.flake8

```diff
@@ -2,6 +2,14 @@
 # W191 - Tabs over spaces
```

Comment:
There's a module called flake8-tabs that I think augments flake8. I'll look into setting it up.

Reply:
I think this will be part of a separate PR though. I also want to setup the [pre-commit](https://pre-commit.com/) framework so that all the developers can have the same pre-commit hooks for natlas. Going to resolve this and we can circle back in a separate PR specifically for linting configuration. This PR is already massive.